packages/@o3r/core/schematics/ng-add/project-setup/index.ts

```diff
@@ -0,0 +1,67 @@
+import {chain, noop, SchematicContext, Tree} from '@angular-devkit/schematics';
+import * as fs from 'node:fs';
+import * as path from 'node:path';
+import {NgAddSchematicsSchema} from '../schema';
+
+/**
+ * Enable all the otter features requested by the user
+ * Install all the related dependencies and import the features inside the application
+ *
+ * @param options
+ */
+export const prepareProject = (options: NgAddSchematicsSchema, rootFolder: string) => async (tree: Tree, context: SchematicContext) => {
+  const {
+    createAzurePipeline, generateRenovateConfig, o3rBasicUpdates, updateAdditionalModules, updateCmsAdapter,
+    updateCustomizationEnvironment, updateFixtureConfig, updateLinter,
+    updateOtterEnvironmentAdapter, updatePlaywright, updateStore
+  } = await import('../../rule-factories/index');
+
+  const corePackageJsonContent = JSON.parse(fs.readFileSync(path.resolve(rootFolder, '..', '..', 'package.json'), {encoding: 'utf-8'}));
```

Comment:
same, you could use the tree here:
`JSON.parse(tree.readContent(packageJsonPath)`